frontend/src/app/modules/common/notifications/upload-progress.component.ts

```diff
@@ -26,16 +26,12 @@
 // See docs/COPYRIGHT.rdoc for more details.
 //++
 
-import {Component, EventEmitter, Input, OnDestroy, OnInit, Output} from '@angular/core';
-import {
-  UploadFile,
-  UploadHttpEvent,
-  UploadInProgress
-} from "core-components/api/op-file-upload/op-file-upload.service";
-import {untilComponentDestroyed} from "ng2-rx-componentdestroyed";
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {UploadFile, UploadHttpEvent, UploadInProgress} from "core-components/api/op-file-upload/op-file-upload.service";
 import {HttpErrorResponse, HttpEventType, HttpProgressEvent} from "@angular/common/http";
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
 import {debugLog} from "core-app/helpers/debug_output";
+import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/85e9c35a9c9361e0038637188a645cba25065d78/frontend/src/app/modules/common/notifications/upload-progress.component.ts#L35"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>